POEApi.Model/ProxyMapper.cs

```diff
@@ -298,5 +299,18 @@ internal static List<Tab> GetTabs(List<JSONProxy.Tab> tabs)
                 throw;
             }
         }
+
+        public static ChargeInfo GetCharges(List<JSONProxy.Property> list)
```

Comment:
Put a try catch around this method, given how frequently GGG can change things it's safer to catch the exception, log it and return a default value.